site/using-weave/weave-cloud.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Using Weave Cloud to get started with Weave Net
+menu_order: 10
+---
+
+Weave Net provides a simple to deploy networking solution for containerized apps. Here, we describe how to manage a Weave container network using a sample application which consists of two simple `netcat` services deployed to containers on two separate hosts.
+
+This section contains the following topics: 
+
+ * [Launching Weave Net](#launching)
+ * [Creating Peer Connections Between Hosts](#peer-connections)
+ * [Testing Container Communications](#testing)
+ * [Starting the Netcat Service](#start-netcat)
+
+
+###<a name="launching"></a>Signing in to Weave Cloud
```

Comment:
A whitespace should be added for titles.
Jekyll, for the new website, apparently does not like them w/o whitespaces (see: #2889).